packages/api-clients/open-api/agreementApi.yml

```
@@ -847,6 +890,151 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  "/agreements/verify":
```

just by seeing the path it is not clear what the endpoint does.
What about
```suggestion
  "/tenants/{tenantId}/eservices/{eserviceId}/descriptors/{descriptorId}/certifiedAttributes/validate":
```
I also removed the `verify` keyword that could be misleading in the context of attributes.

This should be a `GET` since no data is modified